src/Microsoft.AspNet.Mvc.Core/Infrastructure/ObjectResultExecutor.cs

```diff
@@ -115,15 +115,13 @@ public virtual Task ExecuteAsync(ActionContext context, ObjectResult result)
             if (selectedFormatter == null)
             {
                 // No formatter supports this.
-                Logger.LogWarning("No output formatter was found to write the response.");
+                Logger.NoFormatter(formatterContext.ContentType);
```

Comment:
Same feedback, pass things in the formatter, then it's more resilient to change if we need to add more details to the logging.